package/MDAnalysis/analysis/rms.py

```diff
@@ -138,70 +137,90 @@
 logger = logging.getLogger('MDAnalysis.analysis.rmsd')
 
 
-def rmsd(a, b, weights=None, center=False):
+def rmsd(a, b, weights=None, center=True, superposition=False):
```

Comment:
Then we and new users fall into the `superposition` trap again. For the center I accept the backwards compatibility argument.